Makefile.am

```diff
@@ -151,12 +151,12 @@ clean-venv:
 	fi
 
 .PHONY: tests
-tests: tox
-	tox -e "tests,lib,network,virt,gluster,hooks"
+tests: tox venv
+	. $(VENV)/activate && tox -e "tests,lib,network,virt,gluster,hooks"
 
 .PHONY: tests-storage
-tests-storage: tox
-	tox -e "storage"
+tests-storage: tox venv storage
+	. $(VENV)/activate && tox -e "storage"
```

Comment:
same

Reply:
What about the dependency with storage? That one is still valid I think.